packages/cardano-services/src/Program/loadHttpServer.ts

```diff
@@ -128,7 +129,10 @@ const serviceMapFactory = (options: ServiceMapFactoryOptions) => {
       if (!genesisData)
         throw new MissingProgramOption(ServiceNames.StakePool, ProgramOptionDescriptions.CardanoNodeConfigPath);
       const stakePoolProvider = new DbSyncStakePoolProvider(
-        { paginationPageSizeLimit: args.options!.paginationPageSizeLimit! },
+        {
+          paginationPageSizeLimit: args.options!.paginationPageSizeLimit!,
+          responseConfig: { search: { metrics: { apy: !args.options?.disableStakePoolMetricApy } } }
```

Comment:
I think the assignment is wrong here since `disableStakePoolMetricApy` is a boolean with a negative name and [default value](https://github.com/input-output-hk/cardano-js-sdk/pull/609/files#diff-5407e26c5bfc8209e2abcdcca73de87a43df7d6e640d4f64ad3124684bf5d665R13) of `false` if not specified, therefore that will skip over [this condition](https://github.com/input-output-hk/cardano-js-sdk/pull/609/files#diff-3a938d2c168d4b251612c6efabf201ac70e7585c3c1c2149b3c34e71981e6666R236-R243) but in reality with `disableStakePoolMetricApy: false` we want to perform the query and return the APY, correct?
@rhyslbw

Reply:
Maybe you missed that it's inverted? `!args.options?.disableStakePoolMetricApy`

Reply:
Ah yeah, I saw it initially like `args.options?.disableStakePoolMetricApy!` as always being presented for the ts :D
Omit my statement, seems my eyes are tired lol @rhyslbw